### ISSUE TITLE
Cannot ANALYZE table with column called `totalrows`

### ISSUE THREAD

**Issue by hlinnaka** [greenplum-db/gpdb#9511]
### Greenplum version or build
6X_STABLE, master

### Step to reproduce the behavior
```
postgres=# create table xx (totalrows int4);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'totalrows' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
CREATE TABLE
postgres=# analyze xx;
ERROR:  column name "totalrows" specified more than once  (seg0 127.0.0.1:40000 pid=31593)
```
This is because the gp_acquire_sample_rows() function we use internally to collect the sample contains an internal column called "totalrows", and it clashes with the actual table column. Same problem with `totaldeadrows` and `oversized_cols_bitmap`

**Comment by ashwinstar** [comment]
Does this PR https://github.com/greenplum-db/gpdb/pull/9464 implicitly fixes this problem as well?

**Comment by zhangh43** [comment]
Yes, That PR could fix this problem.
 PR #9464 is for 6X, and I closed it and open PR #9522 to fix it on master

**Comment by zhangh43** [comment]
Issue fixed in #9522
